.github/workflows/aws-native-weekly-sdk-generation.yml

```diff
@@ -87,34 +75,23 @@ jobs:
       id: vars
       run: echo commit-hash="$(git rev-parse HEAD)" >> "$GITHUB_OUTPUT"
       working-directory: aws-cloudformation-user-guide
-    - name: Commit changes
-      run: >-
-        git add sdk
-
-        git commit -m "Regenerating SDKs based on aws-cloudformation-user-guide @ ${{ steps.vars.outputs.commit-hash }}" || echo "ignore commit failure, may be empty"
-
-        git add .
-
-        git commit -m "Regenerating based on aws-cloudformation-user-guide @ ${{ steps.vars.outputs.commit-hash }}" || echo "ignore commit failure, may be empty"
-
-        git push origin generate-sdk/${{ github.run_id }}-${{ github.run_number }}
     - name: Create PR
       id: create-pr
-      uses: repo-sync/pull-request@7e79a9f5dc3ad0ce53138f01df2fad14a04831c5 # v2.12.1
+      uses: peter-evans/create-pull-request@v8
```

Comment:
Heads up: This may interfere with `checkout` action above.

See https://github.com/actions/checkout/issues/2299

And, also, worth checking with @blampe : https://github.com/pulumi/ci-mgmt/issues/1946#issue-3748167994